igvc_navigation/src/mapper/main.cpp

```diff
@@ -208,8 +218,10 @@ int main(int argc, char **argv)
   pNh.getParam("occupancy_grid_length", length_y);
   pNh.getParam("occupancy_grid_width", width_x);
   pNh.getParam("occupancy_grid_resolution", resolution);
+  pNh.getParam("occupancy_grid_threshold", occupancy_grid_threshold);
   pNh.getParam("start_X", cont_start_x);
   pNh.getParam("start_Y", cont_start_y);
+  pNh.getParam("increment_step", increment_step);
   pNh.getParam("orientation", orientation);
```

Comment:
Can I ask what you mean by when you say that it is set in the callback? Do you mean that this should be set in the callback? This variable used to be just a magic number, and I wasn't sure where would be the best place to define it.

Reply:
It's not a magic number, it is set in the callback, line 69. In fact this should never run set to zero since that would assume that we are facing east without and input as to why that is.

Reply:
Oh, I thought you were referring to `increment_step`, which was why I was confused. You are referring to `orientation` here right?

Reply:
Nope, the bottom line is the line I am commenting on